### ISSUE TITLE
this program permanently changed my controller's settings even when I don't have it active

### ISSUE THREAD

**Issue by GaDemICE** [x360ce/x360ce#1393]
I looked for this program so that Ryujinx could recognize my gamepad.  That worked.
The next day i opened my PC and went to play some other games like Hollow Knight and Ghost Song.
I didn't run x360ce.
Ghost Song: My button inputs were all ways of swapped. X was Y, start and back were on the back-triggers,  and no button seemed to correspond to what would have been the shoulder buttons.  That game became unplayable.
Hollow Knight: The game didn't recognize the controller at all, when it used to. 
Axiom Verge: I immediately tested this game afterwards since i finished it with said controller last week and it also didn't recognize my controller.
Yuzu: I decided to test that one last. Suprisingly this one still worked. Properly.

Now i go and run x360ce.
Ghost Song: It is now running BOTH the button inputs i had mapped my controller with via x360ce,  AND the messed up ones from before.  So for example when i press the left trigger button that should be the aim-mode,  it first opens the character menu (which normally is opened by start, as mentioned before), and after i close that it goes into aim-mode.  Or it jumps and shoots with the same button.
Hollow Knight: the game now recognizes my controller, but is occasionally stopping accepting my inputs for some seconds.  That wasn't happening before.
Axiom verge: This one seems to work properly.
Yuzu: still works properly.

So basically:
I have lost the ability to play some games with controler,  with or without x360ce
I have become entirely dependent on having to run x360ce for some games,  and some of those don't even  work flawlessly.

I have tried uninstalling the virtual xbox360 device that is created in device manager,  but to no avail.
I have not been able to locate my actual controller in the device manager.
I don't have the knowledge to search the registry for whatever has caused this permanent change in the controller settings so i can revert it.

Please tell me there is something I can do to save my self from a PC format...

**Comment by Squall-Leonhart** [comment]
lol, no.
